docs/source/broker/run-a-broker-instance/initial-setup.rst

```diff
@@ -3,9 +3,17 @@ Initial Setup
 
```

Comment:
You tell the user to set `PROJECT_ID` and `KEY_PATH` but only use those variables once. It would be clearer to have them directly set `GOOGLE_CLOUD_PROJECT` and `GOOGLE_APPLICATION_CREDENTIALS`

Reply:
✅